### PR TITLE
[#2259] View project source buttons. Yay.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: src/editor/share-editor.js [hunk 1]
```diff
@@ -119,10 +121,12 @@ define([ "editor/editor", "editor/base-editor", "ui/user-data",
 
         projectURL.value = e.url;
         previewBtn.href = e.url;
+        embedUrl = e.url.replace( "/v/", "/e/" );
```
Comment (humphd) [inline src/editor/share-editor.js]: This is going to keep breaking.  We need a way to ask the server for the embed URL for a given project ID.  I had to change our URL structure in the S3 ticket.

File: src/ui/header.js [hunk 2]
```diff
@@ -63,6 +64,8 @@ define([ "dialog/dialog", "util/lang", "ui/user-data", "ui/widget/tooltip" ],
           butter.editor.openEditor( "share-properties" );
           _previewBtn.classList.remove( "butter-hidden" );
           _previewBtn.href = e.url;
+          _viewSourceBtn.classList.remove( "butter-hidden" );
+          _viewSourceBtn.href = "view-source:" + e.url.replace( "/v/", "/e/" );
```
Comment (humphd) [inline src/ui/header.js]: Same deal here.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e55fe13fa5a4437/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/f34b0e4452b0a27/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/e55fe13fa5a4437/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/f34b0e4452b0a27/output.txt

Total script time: 3.00 mins

File: src/editor/share-editor.js (hunk 1 above)
Comment (mjschranz) [inline src/editor/share-editor.js]: I actually do this in ticket #2297 :)

**Comment by mjschranz** [comment]
Landed on master.

**Pull request closed without merging**